AK/Vector.h

```diff
@@ -874,9 +874,16 @@ requires(!IsRvalueReference<T>) class Vector {
     [[nodiscard]] static ErrorOr<ByteBuffer> create_uninitialized(size_t size)
     requires(IsSame<T, u8>)
     {
-        // FIXME: Optimize for uninitialized memory again
-        return create_zeroed(size);
+        auto buffer = Vector();
+        TRY(buffer.try_ensure_capacity(size));
+        buffer.m_size = size;
+        // FIXME: gcc can see that the allocation is uninitialized, and refuses the call to memmove that happens in move-contruct and move-assign.
+        // Therefore, we initialize the allocation here, despite the name.
+        // This used to be no problem in ByteBuffer, presumably because
```

Comment:
_"presumably because"_...?
If you're talking about the following statement, perhaps put a `:` at the end of the comment... or finish the sentence :^)